recipe/build.sh

```diff
@@ -1,9 +1,8 @@
 mkdir build
 cd build
-cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+cmake ${CMAKE_ARGS} \
+      -DCMAKE_BUILD_TYPE=RelWithDebInfo \
       -DCMAKE_VERBOSE_MAKEFILE=ON \
-      -DCMAKE_INSTALL_PREFIX=$PREFIX \
```

Comment:
I guess the install prefix is now specified by conda-forge setup in CMAKE_ARGS?
It would have been nice to have an explanation in the commit message.

Reply:
Good point. Sorry. I synchronised some things from main at the same time. Should probably have split in several commits.

I thought you would remember https://github.com/conda-forge/cpptango-feedstock/pull/10/files#r963500014 :innocent: I didn't learn :sweat_smile:

Reply:
Ooops! Sorry I have a good short term memory only! 😅 This is why I find it useful to explain these kinds of change in the associated commit messages.